lib/features/map_view/widgets/map_widget.dart

```diff
@@ -38,7 +39,13 @@ class MapWidget<T extends GoogleNavigable> extends ConsumerWidget {
         TileLayer(
           urlTemplate: OpenStreetMapConfig.tileUrl,
           userAgentPackageName: OpenStreetMapConfig.userAgent,
-          tileProvider: CancellableNetworkTileProvider(),
+          tileProvider: CachedTileProvider(
```

Comment:
future note, on web we may want to use CancellableNetworkTileProvider instead of cached one - but only a future note